{"links": {"self": {"href": "data/repositories/ignitionrobotics/ign-gazebo/pullrequests/553/comments/142735232.json"}, "html": {"href": "#!/ignitionrobotics/ign-gazebo/pull-requests/553/_/diff#comment-142735232"}}, "deleted": false, "pullrequest": {"type": "pullrequest", "id": 553, "links": {"self": {"href": "data/repositories/ignitionrobotics/ign-gazebo/pullrequests/553.json"}, "html": {"href": "#!/ignitionrobotics/ign-gazebo/pull-requests/553"}}, "title": "3 to 4 2020-03-10"}, "content": {"raw": "Still getting some failed tests, four of them are for tests that haven\u2019t run and the results are expected and the fifth is a new issue with the PeerTracker, I\u2019ve tried to resolve it here [215d269](#!/ignitionrobotics/ign-gazebo/commits/215d269b582902261bbc6a5ca79a99a83f1ffa8e) along with fixing some warnings that came up", "markup": "markdown", "html": "<p>Still getting some failed tests, four of them are for tests that haven\u2019t run and the results are expected and the fifth is a new issue with the PeerTracker, I\u2019ve tried to resolve it here <a data-is-external-link=\"true\" href=\"#!/ignitionrobotics/ign-gazebo/commits/215d269b582902261bbc6a5ca79a99a83f1ffa8e\" rel=\"nofollow\">215d269</a> along with fixing some warnings that came up</p>", "type": "rendered"}, "created_on": "2020-03-30T22:38:33.971988+00:00", "user": {"display_name": "John Shepherd", "uuid": "{c4803212-effe-4921-9323-02cd01b8bc9c}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bc4803212-effe-4921-9323-02cd01b8bc9c%7D"}, "html": {"href": "https://bitbucket.org/%7Bc4803212-effe-4921-9323-02cd01b8bc9c%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/5d114073a232e50bd6f26983/41da687b-99ee-4c7e-a527-0202b6dc55f5/128"}}, "nickname": "John Shepherd", "type": "user", "account_id": "5d114073a232e50bd6f26983"}, "updated_on": "2020-03-30T22:38:33.974601+00:00", "type": "pullrequest_comment", "id": 142735232}